packages/web-components/fast-foundation/src/scroller/scroller.ts

```diff
@@ -0,0 +1,276 @@
+import { attr, DOM, FASTElement, observable } from "@microsoft/fast-element";
+// TODO: the Resize Observer related files are a temporary stopgap measure until
+// Resize Observer types are pulled into TypeScript, which seems imminent
+// At that point these files should be deleted.
+// https://github.com/microsoft/TypeScript/issues/37861
+import {
+    ConstructibleResizeObserver,
+    ResizeObserverClassDefinition,
+} from "../anchored-region/resize-observer";
+import { ResizeObserverEntry } from "../anchored-region/resize-observer-entry";
+import { ScrollerControlStyles } from "./scroller.styles";
+
+declare global {
+    interface WindowWithResizeObserver extends Window {
+        ResizeObserver: ConstructibleResizeObserver;
+    }
+}
+
+/**
+ * The views types for a scroller {@link @microsoft/fast-foundation#(Scroller:class)}
+ * @public
+ */
+export type ScrollerView = "default" | "mobile";
+
+/**
+ * A Scroller Custom HTML Element
+ * @public
+ */
+export class Scroller extends FASTElement {
+    /**
+     * Reference to DOM element containing the content to scroll
+     * @public
+     */
+    public content: HTMLDivElement;
```

Comment:
Might these be any html element? Is it possible to allow the template to be more flexible?

Reply:
I was too ambiguous here. This isn't the actual content. This is a ref used in the template, it is the content container. I'll update to make it more clear.